iothub_client/samples/pnp/pnp_simple_thermostat/pnp_simple_thermostat.c

```diff
@@ -12,16 +12,17 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
+#include <limits.h>
```

Comment:
>limits [](http://example.com/codeflow?start=10&length=6)

only api that is changed in this file is strtod, which comes from #include <stdlib.h> any reason this lib is added? Is this added for some warning?